src/index.html

```diff
@@ -2,11 +2,325 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="samsung" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+             href="#laptops&Computers"
+             class="nav__link"
+             data-qa="nav-hover"
+            >
+             Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container">
+        <div class="card" data-qa="card">
+          <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+            class="card__image"
+      >
```

Comment:
```suggestion
          <img
            src="./images/imac.jpeg"
            alt="APPLE A1419 iMac 27"
            class="card__image"
          >
```
Pay attention to formatting. Fix all cases